feat: added `op-test-sequencer` support for sysext

sadiq1971 opened the pull request:
Part of #15265
op-test-sequencer has been added to kurtosis [here](https://github.com/ethpandaops/optimism-package/pull/368)


### Adds op-test-sequencer integration to sysext

* This update introduces support for op-test-sequencer in the sysext orchestrator.
* Dynamically detects and initializes test sequencers from L2 service definitions.
* Deduplicates sequencer instances across multiple L2 chains.
* Configures RPC clients for each chain with JWT authentication.
* Registers sequencers with associated control clients for multi-chain sequencing operations.
 
 This enables testing the reorg tests (L1 reorg will still not be supported).

sadiq1971 committed:
added test sequencer support on sysext

sadiq1971 committed:
added l1 control client

sadiq1971 committed:
added todo

nonsense commented:
/ci authorize 06864c5c3d9df0b73e90fb17821faf92f0cb9752

nonsense commented:
@sadiq1971 thank you for this PR. It looks like some of the op-acceptance-tests are assuming exactly one test-sequencer, and need adjustment with these changes.

sadiq1971 commented:
> @sadiq1971 thank you for this PR. It looks like some of the op-acceptance-tests are assuming exactly one test-sequencer, and need adjustment with these changes.

I will check that will place an update

sadiq1971 commented:
@nonsense What do you think will be ideal to update? 
- Updating the assert check to allow any number of test sequencer? Or
- Update the kurtosis devent to include op-test-sequencer?

Update: Actully, did both. Updated `interop.yaml ` and added `test-sequencer` there. Removed the `test-sequencer` from the minimal preset as it was not being used.

sadiq1971 committed:
added test sequencer to interop devnet

sadiq1971 committed:
merged develop

nonsense commented:
/ci authorize 5489d22d6b96aa3aa7a76ee643638b6e202019dc

nonsense reviewed (approved):
lgtm. i didn't realize we are not using the TestSequencer within the minimal preset, so it is fine removing it.

sadiq1971 commented:
@nonsense Can we merge this?
